lib/cc/services/github_pull_requests.rb

```diff
@@ -93,27 +86,34 @@ def update_status_pending
   end
 
   def update_status(state, description)
-    params = {
-      state:       state,
-      description: description,
-      target_url:  @payload["details_url"],
-      context:     "codeclimate"
-    }
-    service_post(status_url, params.to_json)
+    if config.update_status
+      params = {
+        state:       state,
+        description: description,
+        target_url:  @payload["details_url"],
+        context:     "codeclimate"
+      }
+      @response = service_post(status_url, params.to_json)
+    end
   end
 
   def add_comment
-    if !comment_present?
-      body = {
-        body: COMMENT_BODY % @payload["compare_url"]
-      }.to_json
-
-      service_post(comments_url, body) do |response|
-        doc = JSON.parse(response.body)
-        { id: doc["id"] }
+    if config.add_comment
+      if !comment_present?
+        body = {
+          body: COMMENT_BODY % @payload["compare_url"]
+        }.to_json
+
+        @response = service_post(comments_url, body) do |response|
+          doc = JSON.parse(response.body)
+          { id: doc["id"] }
+        end
+      else
+        @response = {
+          ok: false,
```

Comment:
This may be a pre-existing problem but I don't think this branch should respond with `ok: false` -- it should be `ok: true`.

My understanding is that an `ok: false` has user facing implications: it's a red stop sign meaning the integration isn't working properly. However, the comment existing on a PR if we've already commented is _expected_. It happens if you open a PR and then push more commits (so a lot).